IntegrationTests/UI.IntegrationTests/CommandsDialogs/FormEditorTests.cs

```diff
@@ -79,12 +70,14 @@ public void HasChanges_updated_correctly()
             }
         }
 
+        [Ignore("Unstable UTF8EncodingSealed result")]
```

Comment:
Perhaps ignore `+UTF8EncodingSealed`?

Reply:
I really would like to know why this appear at this point, also when rolling back the changes...
What exact wording do you want?

Reply:
(Too short as too often) I meant to make the test accept both results.

Reply:
I do not see a way to create this encoding.
Would a regex match on the output encoding text be OK?

Reply:
I withdraw my proposal because I do not know enough about this topic.
BTW: For #9860, a BOM must be added to the `English.xlf`.
Perhaps this is caused by different .NET patch levels of different AppVeyor instances.

Reply:
@RussKie Suggestions how to proceed with these tests?
(of course you are welcome to comment on the rest of the PR too...)

Reply:
> BTW: For #9860, a BOM must be added to the `English.xlf`.
> Perhaps this is caused by different .NET patch levels of different AppVeyor instances.

This sounds like https://github.com/dotnet/runtime/issues/63585